libs/core/src/lib/file-uploader/directives/file-uploader-dragndrop.directive.ts

```diff
@@ -0,0 +1,117 @@
+import { Directive, EventEmitter, HostListener, Input, Output } from '@angular/core';
+
+/**
+ * Directive that handles the drag and drop feature of the file input.
+ */
+@Directive({
+    selector: '[fdFileDragnDrop]'
+})
+export class FileUploaderDragndropDirective {
+    /** Whether multiple files can be dropped at once. */
+    @Input()
+    multiple: boolean = true;
```

Comment:
have you run linter? Seems like strict typing enforcement has been removed recently so types may not be needed.

Reply:
yes fixed all lint errors